src/users/containers/utils/user-roles-util.ts

```diff
@@ -18,23 +18,25 @@ export class UserRolesUtil {
         return roles;
         }
 
-    static getRolesDeleted(user: any, permissions: string[]): any[] {
+    public static getRolesDeleted(user: any, permissions: string[]): any[] {
         const roles = [];
-        user.roles.forEach( (permission) => {
-            if (!permissions.includes(permission) && !AppConstants.CCD_ROLES.includes(permission)) {
-              roles.push({
-                  name: permission
-              });
-              if (permission === 'pui-case-manager') {
-                const ccdRolesTobeRemoved = UserRolesUtil.GetRemovableRolesForUser(user, AppConstants.CCD_ROLES);
-                ccdRolesTobeRemoved.forEach(newRole => roles.push(newRole));
+        if (user.roles) {
+          user.roles.forEach( (permission) => {
+              if (!permissions.includes(permission) && !AppConstants.CCD_ROLES.includes(permission)) {
+                roles.push({
+                    name: permission
+                });
+                if (permission === 'pui-case-manager') {
```

Comment:
Not keen on hardcoded strings, surely can be abstracted out into a config/constant etc